docs/competition/team-support/battery-charging.md

```diff
@@ -7,44 +7,77 @@ original:
 
 We (Student Robotics) manage the charging of all batteries at the competition. Battery Charging volunteers are responsible for ensuring that there is a continuous supply of charged batteries available for use by teams.
 
-## Equipment
-
- * Battery chargers
- * Discharged battery box
- * Charged battery box
- * Damaged battery/charger box
- * Red insulating tape
- * Example good batteries (one brand new, one squidgy)
+## Setup
+
+### Equipment
+Most batteries and chargers will arrive with teams, but any batteries, bags, and chargers in inventory should be prepared during setup.
```

Comment:
We should say somewhere that we're explicitly only using the IMAX chargers and what to do with the others (putting them in a box out the way is a valid option).

Reply:
Fixed in 3738771